src/Panel/Graph/GraphPanel.java

```diff
@@ -18,11 +19,14 @@
 
     protected Graphics2D artist;
 
+    protected int label = 25;
```

Comment:
These changes should be in a separate branch since they are not related to the menu bar.